### DISCUSSION TITLE
微博扫码登录报错

### DISCUSSION THREAD

**Discussion by gumpBOBO** [walinejs/waline#1249] [Q&A] [answered]
返回信息：
{"errno":400,"message":"400 - {\"error\":\"invalid_grant\",\"error_code\":21325,\"request\":\"/oauth2/access_token\",\"error_uri\":\"/oauth2/access_token\",\"error_description\":\"invalid authorization code:0a9f16d90f5e2cfe9bdc765e3247db69\"}"}

版本：2.6.1
用的国际版，博客放在vercel

**Comment by lizheming** [comment, marked as answer]
是我这边申请的微博应用审核失败的问题，已经重新审核了

**Reply by Likenttt** [reply]
刚刚发现还是400的报错. 请问这个资质能否自行维护, 避免公用的导致类似的问题

**Reply by lizheming** [reply]
https://waline.js.org/reference/env.html#%E9%AB%98%E7%BA%A7%E9%85%8D%E7%BD%AE 看下文档
